src/main/java/com/simple/metrics/kafka/DropwizardReporter.java

```diff
@@ -0,0 +1,86 @@
+package com.simple.metrics.kafka;
```

Comment:
This seems like a reasonable package name, to me!

> Is that considered bad form to patch new classes into an existing package you don't own, or is that okay? Would it be more natural to users of this package to access the class as `org.apache.kafka.common.metrics.DropwizardReporter`?

Not sure whether it's "considered bad form," but I would find it potentially confusing and would give some lengthy side-eye. 😉